### DISCUSSION TITLE
/blog/2021/08/21/r2-euler/

### DISCUSSION THREAD

**Discussion by andrewheiss** [andrewheiss/ath-quarto#40] [Blog comments]
Exploring R² and regression variance with Euler/Venn diagrams | Andrew Heiss 
 https://www.andrewheiss.com/blog/2021/08/21/r2-euler/

**Comment by andrewheiss** [comment]
```header
Originally posted by Sanghoon Park on 2021-08-24 01:53:52
````
These are the great and intuitive visualizations to understand the logic
of regression and collinearity. Thanks for the post!

**Comment by andrewheiss** [comment]
```header
Originally posted by Claus on 2022-03-15 11:13:33
````
This is great and what I have been looking for (also for teaching). I
sometimes wonder whether squares would be easier to understand (because
the edge length could be the empirical standard deviation?) but I
presume overlapping those looks messy. Thanks!

**Comment by magnuson8** [comment]
Thanks.  This is helpful at the same time to both novices and to experts.

**Comment by stevenlijunxian** [comment]
Really appreciate your clear illustration! I followed your R-code and fully understood the whole process!!! Thank you sooooo much!!!!
